azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherActivity.kt

```diff
@@ -27,7 +27,9 @@ class CallLauncherActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         // Register Memory Viewer with FeatureFlags
-        initializeMemoryViewFeature(this)
+        conditionallyRegisterDiagnostics(this)
+        FeatureFlags.registerAdditionalFeature(AdditionalFeatures.secondaryThemeFeature)
+
         // Initialize the FeatureFlags enum
```

Comment:
Adam, I think we do nor need code comments as code is very clear? please share your thoughts.